IssueTrackerApplication/Services/UserService.cs

```diff
@@ -11,11 +11,19 @@ public UserService(IGenericRepository<User> _repository, IMapper _mapper) : base
         {
         }
 
-        public async Task<bool> LoginUser(CreateUserCommand userCommand)
+        public async Task<string> LoginUserAsync(CreateUserCommand userCommand)
         {
             var user = _mapper.Map<User>(userCommand);
 
-            return await _repository.ExistsWithConditionAsync(anyUser => anyUser.Name == user.Name && anyUser.Email == user.Email);
+            var userExists = await _repository
+                .GetUniqueWithConditionAsync(anyUser => anyUser.Name == user.Name && anyUser.Email == user.Email);
+
+            if (userExists == null)
+            {
+                return null;
+            }
+
+            return userExists.Role;
```

Comment:
```suggestion
            return userExists?.Role;
```

this way you can get rid of the `if ( == null)` check

Reply:
Added this change to the latest commit.